### PR TITLE
Fixed some weird translations (different spech context in menu)

### PR DESCRIPTION
**Pull request by SrRapero720**
After see traslation i found some translations "weird" because spech context is not the specified in property name, so i made a few changes

Just to add, property "manager.editor.option.visibility.keep.info" is never displayed on launcher

### CODE REVIEW

**Review by SrRapero720 — COMMENTED** [review]
Ñ

File: assets/launcher/lang/sklmessages_es_MX.properties [hunk 1]
```diff
@@ -70,36 +70,36 @@ about.translation.authors=SrRapero720
...
-login.button.type.offline=Acceder Offline
+login.button.type.offline=Iniciar Sesion
 login.label.username=Usuario
-login.dropdown.name=Usuarios con Session activa
+login.dropdown.name=Sesiones activas
 login.button.play=Jugar
 login.button.logout=Cerrar session
-login.button.switch.offline=Cambiar a modo Offline
-login.button.switch.online=Cambiar a modo Online
+login.button.switch.offline=Modo offline (sin cuenta)
```
Comment (skmedix) [inline assets/launcher/lang/sklmessages_es_MX.properties:78]: Keep the original meaning of the string, do not add any additional brackets or your own explanation of the string.

File: assets/launcher/lang/sklmessages_es_MX.properties [hunk 2]
```diff
@@ -70,36 +70,36 @@ about.translation.authors=SrRapero720
...
+login.button.switch.online=Modo Online (microsoft)
```
Comment (skmedix) [inline assets/launcher/lang/sklmessages_es_MX.properties:79]: Keep the original meaning of the string, do not add any additional brackets or your own explanation of the string

File: assets/launcher/lang/sklmessages_es_MX.properties [hunk 3]
```diff
@@ -112,25 +112,25 @@ manager.profile.menu.duplicate=Duplicar
...
 manager.editor.header.title=Editar Instancia
-manager.editor.footer.button.save=Guardar
 manager.editor.option.name.name=Nombre de la Instancia
 manager.editor.option.name.placeholder=<Instancia sin nombre>
 manager.editor.option.version.name=Version
 manager.editor.option.directory.name=Carpeta del juego
 manager.editor.option.directory.placeholder=<Carpeta por defecto>
 manager.editor.option.directory.info.label=Algunos mods (no siempre) necesitan ser instalados en la carpeta .minecraft. Para usar la carpeta .minecraft
 manager.editor.option.directory.info.link=Haz click aqui
+manager.editor.footer.button.save=Guardar
```
Comment (skmedix) [inline assets/launcher/lang/sklmessages_es_MX.properties:122]: Don't change the original order of the string declarations

File: assets/launcher/lang/sklmessages_es_MX.properties [hunk 4]
```diff
@@ -112,25 +112,25 @@ manager.profile.menu.duplicate=Duplicar
...
 manager.editor.option.memory.name=Memoria maxima (RAM)
 manager.editor.option.compatibility.name=Modo de compatibilidad
 manager.editor.option.compatibility.enabled=Activado
 manager.editor.option.visibility.name=Visibilidad del launcher
-manager.editor.option.visibility.hide=Esconder y reabrir al cerrar el juego
-manager.editor.option.visibility.close=Cerrar cuando inicie el juego
-manager.editor.option.visibility.keep=Mantener abierto y muestra la consola del juego
-manager.editor.option.visibility.keep.info=Esta opcion no es la adecuada para PC's de bajos recursos
-manager.editor.option.jvmpath.name=Archivo JAVA
+manager.editor.option.visibility.hide=Esconder (hasta cerrar MC)
```
Comment (skmedix) [inline assets/launcher/lang/sklmessages_es_MX.properties:129]: Do not add any additional brackets or your own explanation of the string.

File: assets/launcher/lang/sklmessages_es_MX.properties [hunk 5]
```diff
@@ -112,25 +112,25 @@ manager.profile.menu.duplicate=Duplicar
...
+manager.editor.option.visibility.close=Cerrar completamente
+manager.editor.option.visibility.keep=Mostrar launcher y la consola
+manager.editor.option.visibility.keep.info=NOTA: No apta para computadoras de bajos recursos
```
Comment (skmedix) [inline assets/launcher/lang/sklmessages_es_MX.properties:132]: Original string does not start with any `NOTE:`

File: assets/launcher/lang/sklmessages_es_MX.properties [hunk 6]
```diff
@@ -70,36 +70,36 @@ about.translation.authors=SrRapero720
...
+settings.option.sorting.name=Orden de las Instancias
+settings.option.sorting.bylastplayed.name=Ultima vez jugado
+settings.option.sorting.byname.name=Nombre
 settings.option.default.name=Instancia por defecto
-settings.option.default.hide.name=Ocultar Instancia por defecto
+settings.option.default.hide.name=Ocultar
 settings.option.console.name=Consola del launcher
 settings.option.console.show.name=Mostrar
-settings.about.contributors=Gracias a todos nuestros contribuyentes:
+settings.about.contributors=Gracias a todos los que nos apoyan:
```
Comment (skmedix) [inline assets/launcher/lang/sklmessages_es_MX.properties:99]: I don't think this is correct. Contributor and Supporter mean different things. Contributor is the person who made real changes to the project.

**Review by skmedix — CHANGES_REQUESTED** [review]

File: assets/launcher/lang/sklmessages_es_MX.properties (hunk 6 above)
Comment (SrRapero720) [reply assets/launcher/lang/sklmessages_es_MX.properties:99]: No take literal translate
Apoyar its used to refer to do stuff, contribuir (or contribuyentes) is more for "i donate you money"
when i write that i figure in first instance all on that list are patreons or someting else giving money

Comment (SrRapero720) [reply assets/launcher/lang/sklmessages_es_MX.properties:99]: Take in mind spanish is more "volatil"
for example, "COJER" in spain means "TAKE" but in LatinAmerica means "FUCK" (but less agresive)

Comment (SrRapero720) [reply assets/launcher/lang/sklmessages_es_MX.properties:99]: Also, most google translations are interpretated in a cultural space

File: assets/launcher/lang/sklmessages_es_MX.properties (hunk 5 above)
Comment (SrRapero720) [reply assets/launcher/lang/sklmessages_es_MX.properties]: changed

**Comment by SrRapero720** [comment]
Off topic info: 
I see in discord are complain about "Se ira a la chingada".

Pobably its hard to trust but its wrong translated
image
2023-01-23_13h44_17
2023-01-23_13h47_15

"Vete al carajo" is used when you have enough of a person and want to get out of your space (or when a friend say a terrible idea).
"Vete a la chingada" is more identified like saying "back off" but angry or in a fight.
"Vete a la verga" that is more like saying "go to hell"
"Vete a la mierda" = "Fuck off"

i use "se ira a la chingada" because its the most used expresion to say "everyting is bad" and in some contexts is used like "vete al carajo".  yeah its weird. probably you need to learn more about mexican spanish. its funny see gringo man trying to say some "weird" words

**Comment by SrRapero720** [comment]
clear your Doubts
https://es.wikipedia.org/wiki/La_Chingada#:~:text=Irse%20algo%20a%20la%20Chingada,mierda%20o%20a%20la%20verga%22.

**Comment by SrRapero720** [comment]
I see you hate me, because ignoring me, never ask me for info or context and never check changes, so i will close this pull request. i just give up.

**Pull request closed without merging**